zcash_primitives/src/sapling/verifier/batch.rs

```diff
@@ -68,21 +68,11 @@ impl BatchValidator {
                 *spend.anchor(),
                 &spend.nullifier().0,
                 spend.rk(),
-                &sighash,
-                spend.spend_auth_sig(),
                 zkproof,
                 self,
-                |this, rk, _, spend_auth_sig| {
-                    let rk = redjubjub::VerificationKeyBytes::<redjubjub::SpendAuth>::from(
-                        rk.0.to_bytes(),
-                    );
-                    let spend_auth_sig = {
```

Comment:
This is a nice simplification. The name overloading (while referring to a different type) confused me a bit at the use site.